### PR TITLE
Create CSS tutorial

### PR DESCRIPTION
**Pull request by joachimvh**

### CODE REVIEW

File: css-tutorial.md [hunk 1]
```diff
@@ -0,0 +1,483 @@
...
+In case that you want multiple users to have their WebID on your server,
+you could have chosen to create your WebID in a specific namespace during setup.
+
+#### Access Control List
+Your WebID needs to be publicly accessible, so you can be identified,
+but your other resources can have restricted access.
+For example, when trying to access the profile container http://localhost:3000/profile/
+you will receive a `403: Not Logged In` error.
+
+We will cover how to correctly authenticate in the next chapter,
```
Comment (pheyvaer) [inline css-tutorial.md:240]: Are we working with chapters?

File: css-tutorial.md [hunk 2]
```diff
@@ -0,0 +1,483 @@
...
+The response page will show some extra information about the setup result.
+
+#### WebID
+In the previous step the server created several resources for you,
+including a profile document with a WebID.
+A [WebID](https://dvcs.w3.org/hg/WebID/raw-file/tip/spec/identity-respec.html)
+is a way to identify yourself on the Web.
+We will make use of this a bit later in this tutorial.
+After setup, you can find your profile resource at http://localhost:3000/profile/card ,
+with your WebID being http://localhost:3000/profile/card#me .
```
Comment (pheyvaer) [inline css-tutorial.md:229]: Sometimes you put a space between an url and dot, and sometimes not.

File: css-tutorial.md [hunk 3]
```diff
@@ -0,0 +1,483 @@
...
+more detailed information can be found in the Components.js
+[documentation](http://componentsjs.readthedocs.io/).
+
+### Customizing imports
+The imports you see in the configuration file
+all refer to files found in subfolders of the `config` folder.
+These folders and files have been structured in a such a way
+that every import line corresponds to a specific feature that can be changed.
+
+For example, if you look the config that is used
```
Comment (pheyvaer) [inline css-tutorial.md:425]: "If you look at the config"

File: css-tutorial.md [hunk 4]
```diff
@@ -0,0 +1,483 @@
...
+you can see it has the following import:
+```
+"files-scs:config/storage/backend/memory.json"
+```
+On the other hand, `config/file.json` has the following import instead:
+```
+"files-scs:config/storage/backend/file.json"
+```
+Simply by changing that import in a configuration you can tell it what backend to use.
+Every subfolder of `config` has a `README` document explaining what the available options are.
```
Comment (pheyvaer) [inline css-tutorial.md:436]: `README.md`?

**Review by pheyvaer — APPROVED** [review]
Really great work!!!
Added some minor comments, but over super nice! 🤩

File: css-tutorial.md (hunk 2 above)
Comment (joachimvh) [reply css-tutorial.md:229]: I wasn't sure if github was going to handle it correctly if there was no space. Will see what it did in the cases where I forgot a space.

**Pull request merged**